c7n/resources/asg.py

```diff
@@ -476,6 +476,29 @@ def process(self, asgs, event=None):
                 len(a['Instances']) < a['MinSize']]
 
 
+@filters.register('progagated-tag')
+class PropagatedTagFilter(Filter):
+
+    schema = type_schema(
+        'progagated-tag',
+        key={'type': 'string'},
+        value={'type': 'string'}
+    )
+
+    def process(self, asgs, event=None):
+        key = self.data.get('key', DEFAULT_TAG)
+        value = self.data.get('value', None)
+        results = []
+        for r in asgs:
+            tags = r.get('Tags', [])
+            for t in tags:
+                if not t['PropagateAtLaunch']:
```

Comment:
this the aspect they want to check. nutshell they want to ensure that required tags are going to make it to new instances. this needs to be configurable because common case folks want to find some org tag that the asg is not setup to propagate.